.travis.yml

```diff
@@ -13,14 +13,13 @@ before_script:
   - wget http://browscap.org/stream?q=Full_PHP_BrowsCapINI -O /tmp/browscap.ini
   - if [ "`phpenv version-name`" == "5.5" ] || [ "`phpenv version-name`" == "5.6" ]; then echo 'opcache.enable=1' >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini; fi
   - if [ "`phpenv version-name`" == "5.5" ] || [ "`phpenv version-name`" == "5.6" ]; then echo 'opcache.enable_cli=1' >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini; fi
-  - if [ "`phpenv version-name`" != "hhvm" ] && [ "`phpenv version-name`" != "hhvm-nightly" ]; then echo 'browscap = "/tmp/browscap.ini"' >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini; fi
   - composer self-update
   - composer install --dev -o --prefer-source
   - cp /tmp/browscap.ini resources/browscap.ini
 
 script:
-  - vendor/bin/phpunit --coverage-text --coverage-clover=coverage.clover --colors --verbose
-  - vendor/bin/phpcs --standard=psr2 src
+  - php -d zend.enable_gc=0 -d browscap=resources/browscap.ini vendor/bin/phpunit --coverage-text --coverage-clover=coverage.clover --colors --verbose
```

Comment:
What benefits does disabling GC give us? I know this helped Composer somewhat, but should we be doing this?